index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+<section class="toolbar">
+  <nav class="toolbar logo"><img src="images/logo.png" alt=""></nav>
+  <nav class="toolbar menu"><img src="images/menu.png" alt=""></nav>
+</section>
+<section class ="container">
+  <div class="bg-image1"></div>
+  <div class="bg-image2"></div>
+  <div class="bg-image3"></div>
+  <div class="bg-image4"></div>
+  <div class="bg-image6"></div>
+  <div class="bg-image7"></div>
+  <div class="bg-imagea"></div>
+  <div class="bg-imageb"></div>
+     <div class="container intro"><h1>I'm Victor,
+       Glad to see you!</h1></div>
+          <div class="container content">
+           <p>Hello! I am a software developer! I can help you build a product, feature or website.
+           Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
+          </div>
+     <h4 class="container social">LET'S CONNECT</h4>
+     <ul>
+       <li class="container linked"><img src="images/linkedin.png" alt=""></li>
+       <li class="container twit"><img src="images/twitter.svg" alt=""></li>
+       <li class="container vectorm"><img src="images/vectorm.svg" alt=""></li>
+       <li class="container vectorp"><img src="images/vectorp.svg" alt=""></li>
+       <li class="container vector"><img src="images/vector.svg" alt=""></li>
```

Comment:
- [ ] [OPTIONAL] please try icons should be clickable so it would be better if you put them inside the `<a>` tag